src/main/java/Duke.java

```diff
@@ -1,10 +1,531 @@
+        Random rand = new Random(System.currentTimeMillis());
+        System.out.println("Hello from" + NEWLINE + logos[Math.abs(rand.nextInt() % 4)] + NEWLINE + VERSION);
+    }
+
+    // prints the line divider
+    public static void printDivider() {
+        System.out.println(SPARKLY_TEXT_DIVIDER);
+    }
+
+    // prints the hello when starting
+    public static void printHelloMessage() {
+        String helloGreeting = "Hello! I'm Tootie!" + NEWLINE + "What can I do for you?" + NEWLINE;
+        printDivider();
+        System.out.print(helloGreeting);
+        printDivider();
+    }
+
+    // prints farewell message
+    public static void printFarewellMessage() {
+        String farewellGreeting = "Bye! Hope to see you again soon! " + FLOWER_SMILE_EMOTICON + NEWLINE;
+        System.out.print(farewellGreeting);
+    }
+
+    // prints all list items with index and check
+    public static void printAllTasks(ArrayList<Task> taskList) {
+        if (numTasks == 0) {
+            System.out.println("No tasks found! " + HAPPY_EMOTICON);
+        } else {
+            for (int i = 0; i < numTasks; i++) {
+                System.out.print((i + 1) + ". ");
+                taskList.get(i).printTaskType();
+                if (taskList.get(i).isComplete()) {
+                    System.out.print(TICK_SYMBOL + " ");
+                } else {
+                    System.out.print(CROSS_SYMBOL + " ");
+                }
+                taskList.get(i).printTaskDescription();
+            }
+        }
+
+        // TODO: print "all done ʕ•ᴥ•ʔ" if all tasks done for now
+    }
+
+    // print the message when command is not understood
+    private static void printConfusedMessage() {
+        System.out.println("Command not found? " + CONFUSED_EMOTICON);
+        System.out.println("Type \"help\" for a list of commands!");
+    }
+
+    // print list of commands and example usage
+    private static void printHelpInfo() {
+        System.out.println("Here is the list of commands I understand:" + NEWLINE + NEWLINE + HELP_COMMAND_TEXT);
+    }
+
+    // figure out the command type from userInput
+    private static CommandType extractCommandType(String userInput) {
+        if (userInput.trim().startsWith("help")){
+            return CommandType.HELP;
+        } else if (userInput.trim().startsWith("todo")){
+            return CommandType.ADD_TODO;
+        } else if (userInput.trim().startsWith("deadline")){
+            return CommandType.ADD_DEADLINE;
+        } else if (userInput.trim().startsWith("event")){
+            return CommandType.ADD_EVENT;
+        } else if (userInput.trim().startsWith("list")){
+            return CommandType.LIST;
+        } else if (userInput.trim().startsWith("done")){
+            return CommandType.DONE;
+        } else if (userInput.trim().startsWith("bye")){
+                return CommandType.BYE;
+        } else {
+            return CommandType.UNRECOGNISED;
+        }
```

Comment:
I think you can consider using switch() instead of multiple if statement to improve readability.